.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,26 @@
+name: Docker image build
+
+on: [push, release]
+
+jobs:
+  docker-build-push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
```

Comment:
Note that [Docker recommends][4] using the Docker Hub only for release images, and keep unversioned “latest” images in the GitHub Container Registry.

[4]: https://www.docker.com/blog/best-practices-for-using-docker-hub-for-ci-cd/

Reply:
Oh, okay.  That makes sense. 

Then on: push action can push to ghcr.io and on: release can push to docker hub (In case of making a docker hub account).